### PR TITLE
ga docs for Kommander 1.2

### PR DESCRIPTION
**Pull request by cmastr**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-72970

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (samvantran) [inline pages/ksphere/kommander/1.2/release-notes/index.md:28]:
```suggestion
| **Minimum**        | 1.16.0  |
```

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (samvantran) [inline pages/ksphere/kommander/1.2/release-notes/index.md:29]:
```suggestion
| **Maximum**        | 1.18.x  |
```

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (samvantran) [inline pages/ksphere/kommander/1.2/release-notes/index.md:30]: Basing it off this: https://github.com/mesosphere/yakcl/blob/v0.6.5/Makefile#L41

```suggestion
| **Default**        | 1.18.8  |
```

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (samvantran) [inline pages/ksphere/kommander/1.2/release-notes/index.md:41]: @drewvanstone @juliangieseke pls have someone review and verify what versions will be used for components. yakcl 0.6.5 is latest but i suspect we will have 1 more release to include for the GA

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (samvantran) [inline pages/ksphere/kommander/1.2/release-notes/index.md:49]: Is this everything? do we want to break these up into component-related updates? or maybe since the list is only 10 bullets long it's probably fine to leave as is.

@drewvanstone can you review? I think your team addressed more improvements than is listed here.

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (samvantran) [reply pages/ksphere/kommander/1.2/release-notes/index.md:28]: @juliangieseke should we update this? (1.2 branch) https://github.com/mesosphere/kubeaddons-kommander/blob/1.2.x/addons/kommander/1.2/kommander.yaml#L28

**Review by samvantran — CHANGES_REQUESTED** [review]
Need to double check on versions but rest lgtm

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (juliangieseke) [reply pages/ksphere/kommander/1.2/release-notes/index.md:28]: Im not even sure we still need that. IIRC it comes from times before we had the utility server 🙈

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (gracedo) [inline pages/ksphere/kommander/1.2/release-notes/index.md:43]: for these parent charts (kommander-karma, kommander-thanos, kubecost) do we want to also/instead include the actual component versions (karma, thanos, kubecost) which is probably the more useful information?

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (gracedo) [inline pages/ksphere/kommander/1.2/release-notes/index.md:38]: need to update this with final addon version

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (gracedo) [inline pages/ksphere/kommander/1.2/release-notes/index.md:39]: same, need to update with final chart version

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (samvantran) [reply pages/ksphere/kommander/1.2/release-notes/index.md:28]: probably safer to leave it and just state 1.16 to avoid confusion and then take it out after GA if we really don't need it

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (samvantran) [reply pages/ksphere/kommander/1.2/release-notes/index.md:43]: good question, i vote for actual version (appversion?) but i leave it to yall to decide

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (juliangieseke) [reply pages/ksphere/kommander/1.2/release-notes/index.md:28]: 1.2.x branch is updated with 1.16.0 👍

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (juliangieseke) [reply pages/ksphere/kommander/1.2/release-notes/index.md:41]: yeah, I will update this once we actually tagged the release.

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (juliangieseke) [reply pages/ksphere/kommander/1.2/release-notes/index.md:43]: hm, main idea behind these component was to guide the user (us) to the next thing to check. if we'd have `karma 1.2.3` here, we'd not know which chart version got included and it makes it harder to debug IMHO.

that said, maybe having both would be helpful? not sure how deep we should go tho, I feel like having dep->dep->dep is kinda useless at some point.

Comment (cmastr) [reply pages/ksphere/kommander/1.2/release-notes/index.md:43]: I also prefer the most current version of all components, not just those that have been updated. We try to do this with DCOS as well, and wnat to do that for Konvoy. Consistency would be best.

Comment (cmastr) [reply pages/ksphere/kommander/1.2/release-notes/index.md:43]: ooh, I like the idea of having both. list all of them and figure out a way to format the updated ones differently?

Comment (cmastr) [reply pages/ksphere/kommander/1.2/release-notes/index.md:43]: Could we do this for Konvoy too? Or is it too late?

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (cneth) [inline pages/ksphere/kommander/1.2/release-notes/index.md:60]:
```suggestion
- Fixed an issue that prevented deleting cluster labels.
```

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (cneth) [inline pages/ksphere/kommander/1.2/release-notes/index.md:59]:
```suggestion
- Fixed an issue where the catalog API occasionally returned 500.
```

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (cneth) [inline pages/ksphere/kommander/1.2/release-notes/index.md:61]: This is very unclear, and probably important.  Does the following capture the issue?

```suggestion
- Fixed an issue that erroneously counted the self-attached host cluster against the license count.
```

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (cneth) [inline pages/ksphere/kommander/1.2/release-notes/index.md:62]: I am not sure the 'size' of the bug is relevant here.   How about this:
```suggestion
- Numerous UX Bugfixes and Improvements.
```

**Review by cneth — CHANGES_REQUESTED** [review]

**Review by russurquhart1 — APPROVED** [review]

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (msdolbey) [reply pages/ksphere/kommander/1.2/release-notes/index.md:43]: Couldn't we do it for Konvoy? I feel like it's not too late. Can always add to it/improve the docs.

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (msdolbey) [reply pages/ksphere/kommander/1.2/release-notes/index.md:60]: comitting this - makes sense to me

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (msdolbey) [reply pages/ksphere/kommander/1.2/release-notes/index.md:61]: @cmastr @russurquhart1 you good with that? ☝️

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (msdolbey) [reply pages/ksphere/kommander/1.2/release-notes/index.md:62]: I dig it. Thanks @cneth

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (msdolbey) [inline pages/ksphere/kommander/1.2/release-notes/index.md:62]:
```suggestion
- Numerous UX Bug fixes and Improvements.
```

File: pages/ksphere/kommander/1.2/install-airgapped/index.md
Comment (msdolbey) [inline pages/ksphere/kommander/1.2/install-airgapped/index.md:7]: Like so, @samvantran @cmastr ?

```suggestion
beta: true
```

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (msdolbey) [inline pages/ksphere/kommander/1.2/release-notes/index.md:14]: This still valid?

File: pages/ksphere/kommander/1.2/Architecture/index.md
Comment (msdolbey) [inline pages/ksphere/kommander/1.2/Architecture/index.md:1]: This is not related to this PR, but why is the Architecture capitalized in the URL path here? Everybody good with me changing this to lower case?

cc: @cmastr @russurquhart1 @pierrebeitz

**Review by msdolbey — APPROVED** [review]
I'm good with this whenever we're all ready to pull the trigger
👍 
Pull the trigger

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (gracedo) [inline pages/ksphere/kommander/1.2/release-notes/index.md:39]:
```suggestion
- Addon: 1.2.0-36
```

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (gracedo) [inline pages/ksphere/kommander/1.2/release-notes/index.md:40]:
```suggestion
- Chart: 0.12.16
```

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (gracedo) [inline pages/ksphere/kommander/1.2/release-notes/index.md:41]:
```suggestion
- kommander-federation (yakcl): 0.6.9
```

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (gracedo) [inline pages/ksphere/kommander/1.2/release-notes/index.md:42]:
```suggestion
- kommander-licensing (yakcl): 0.6.9
```

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (gracedo) [reply pages/ksphere/kommander/1.2/release-notes/index.md:50]: Maybe missing:
```
- Updated cert-manager resources to v1
- Added automatic creation of KommanderProjectRole and VirtualGroupKommanderProjectRoleBinding objects based on Workspace objects
- Fixed access to kubecost UI for some cluster roles
- Created the serviceaccount needed for deletion jobs on delete (#220)
- Added --ignore-not-found to resource cleanup jobs to ignore errors if resource has already been deleted
- Fixed metric collection for kubefed, kommander-federation, and kommander-licensing
```

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (gracedo) [reply pages/ksphere/kommander/1.2/release-notes/index.md:44]: App versions in case we want to include them:
```
- karma: 0.70
- thanos: 0.10.1
- cost-analyzer: 1.69.0
- grafana: 6.6.0
```

File: pages/ksphere/kommander/1.2/release-notes/index.md
Comment (gracedo) [reply pages/ksphere/kommander/1.2/release-notes/index.md:14]: I just cut GA tags so.. maybe `16, November 2020` instead?

File: pages/ksphere/kommander/1.2/Architecture/index.md
Comment (cmastr) [reply pages/ksphere/kommander/1.2/Architecture/index.md:1]: can't do this now @msdolbey. It'll break links. But let's open a JIRA please?

**Pull request merged**
